core/core-configuration-layer.el

```diff
@@ -62,6 +62,10 @@
   (expand-file-name (concat user-emacs-directory "private/"))
   "Spacemacs private layers base directory.")
 
+(defconst configuration-certs-directory
```

Comment:
Should be `configuration-layer-certs-directory` ?
I would prefer no abbreviation though even if the variable name is long like `configuration-layer-certificates-directory`.
Also since it is a constant you can make it "private" with a double dash `configuration-layer--certificates-directory`.